.env.test

```diff
@@ -64,14 +62,17 @@ L1_CORE_CONTRACT_ADDRESS="0xE2Bb56ee936fd6433DC0F6e7e3b8365C906AA057"
 RPC_FOR_SNOS=""
 
 
-##### STARKNET SETTLEMENT TEST #####
+##### STARKNET SETTLEMENT #####
 STARKNET_PRIVATE_KEY=0x76f2ccdb23f29bc7b69278e947c01c6160a31cf02c19d06d0f6e5ab1d768b86
 STARKNET_ACCOUNT_ADDRESS=0x3bb306a004034dba19e6cf7b161e7a4fef64bc1078419e8ad1876192f0b8cd1
-MADARA_BINARY_PATH="/path/to/madara"
 
 ## Instrumentation
 OTEL_SERVICE_NAME="madara_orchestrator"
 
 ##### Tests #####
 
 STARKNET_OPERATOR_ADDRESS="0x5b98B836969A60FEC50Fa925905Dd1D382a7db43"
+AWS_SNS_ARN_NAME="madara-orchestrator-arn"
+MADARA_BINARY_PATH="/path/to/madara"
+# pick up by AWS sdk
+AWS_ENDPOINT_URL="http://localhost.localstack.cloud:4566"
```

Comment:
ENV : 
- `AWS_ENDPOINT_URL` 
- `MADARA_BINARY_PATH`
- `AWS_SNS_ARN_NAME`
- `STARKNET_OPERATOR_ADDRESS`
are a part of `.env.test` but not found in `.env.example`

Reply:
only used for tests, it's under the tests section